net/net.sh

```diff
@@ -117,6 +125,37 @@ while getopts "h?T:t:o:f:rD:i:" opt; do
       ;;
     esac
     ;;
+  c)
+    getClientTypeAndNum() {
+      if ! [[ $OPTARG == *'='* ]]; then
+        echo "Error: Expecting keypair \"clientType=numClientType\" but got \"$OPTARG\""
+        exit 1
+      fi
+      local keyValue
+      IFS='=' read -ra keyValue <<< "$OPTARG"
+      local clientType=${keyValue[0]}
+      local numClients=${keyValue[1]}
+      re='^[0-9]+$'
+      if ! [[ $numClients =~ $re ]] ; then
+        echo "error: numClientType must be a number but got \"$numClients\""
+        exit 1
+      fi
+      case $clientType in
+        tps)
+          numBenchTpsClients=$numClients
+        ;;
+        exchange)
+          numBenchExchangeClients=$numClients
+        ;;
+        *)
+          echo "Unknown client type: $clientType"
+          exit 1
+          ;;
+      esac
+      unset IFS
```

Comment:
oh no need to `unset IFS` here, the `IFS='=' read ..` command only modifies IFS for that command